runtime/src/bank/bank_hash_details.rs

```diff
@@ -94,6 +95,9 @@ pub struct SlotDetails {
     #[serde(skip_serializing_if = "String::is_empty")]
     #[serde(default)]
     pub last_blockhash: String,
+    #[serde(skip_serializing_if = "String::is_empty")]
+    #[serde(default)]
```

Comment:
Having these tags on every field is a bit messy; it will be better to have something like `BankHashComponentDetails` that is an `Option` in `SlotDetails`. However, out of scope for this PR + I put this one up for PR first incase there is desire to BP this